### ISSUE TITLE
Typo: 'notcomplaint' instead of 'notcompliant'

### ISSUE THREAD

**Issue by bohwaz** [sisimai/p5-Sisimai#546]
https://github.com/sisimai/p5-sisimai/blob/e90952bc7ce0054c9729285df6c985d4bcb4211b/lib/Sisimai/Rhost/YahooInc.pm#L35

**Comment by azumakuniyuki** [comment]
@bohwaz Thanks for the report! I'll fix the issue soon.
```
% find lib t xt -type f -exec grep -i 'notcomplaintrfc' {} +
lib/Sisimai/Rhost/YahooInc.pm:        'notcomplaintrfc' => ['headers are not rfc compliant'],
```

**Comment by azumakuniyuki** [comment]
Fix this issue after merging #547 (it may conflict)

**Comment by azumakuniyuki** [comment]
@bohwaz Fixed and released as v5.1.0p3 :-) https://github.com/sisimai/p5-sisimai/releases/tag/v5.1.0p3

**Comment by bohwaz** [comment]
Thank you :)
